yarn-project/aztec/src/cli/cmds/start_node.ts

```diff
@@ -134,8 +134,9 @@ export async function startNode(
     };
     let account;
     if (!sequencerConfig.publisherPrivateKey || sequencerConfig.publisherPrivateKey === NULL_KEY) {
-      if (sequencerConfig.validatorPrivateKey) {
-        sequencerConfig.publisherPrivateKey = sequencerConfig.validatorPrivateKey as `0x${string}`;
+      if (sequencerConfig.validatorPrivateKeys?.length) {
+        // NOTE: publisherPrivateKey will be removed and we'll have just one private key per validator/sequencer
```

Comment:
@LHerskind

Reply:
#14287

Reply:
ah this comment is outdated after my discussion with @LHerskind & @just-mitch